source/base/data_out_base.cc

```diff
@@ -3948,25 +3791,16 @@ namespace DataOutBase
                 }
 
 
-              // now compute the projection of
-              // the bilinear cell given by the
-              // four vertices and their heights
-              // and write them to a proper
-              // cell object. note that we only
-              // need the first two components
-              // of the projected position for
-              // output, but we need the value
-              // along the line of sight for
-              // sorting the cells for back-to-
-              // front-output
+              // now compute the projection of the bilinear cell given by the
+              // four vertices and their heights and write them to a proper cell
+              // object. note that we only need the first two components of the
+              // projected position for output, but we need the value along the
+              // line of sight for sorting the cells for back-to- front-output
               //
-              // this computation was first written
-              // by Stefan Nauber. please no-one
-              // ask me why it works that way (or
-              // may be not), especially not about
-              // the angles and the sign of
-              // the height field, I don't know
-              // it.
+              // this computation was first written by Stefan Nauber. please
+              // no-one ask me why it works that way (or may be not), especially
+              // not about the angles and the sign of the height field, I don't
+              // know it.
```

Comment:
here and everywhere else, did you just re-flow all of these comments by hand? The patch is hard to read because the real meat is hidden between all of these unrelated changes. If you had a way to break things out and into a separate patch, that would be swell.

Reply:
i will pull-out a few things once I get it working...

Reply:
p.s. 

> re-flow all of these comments by hand?

no, clang-format

Reply:
I must have not paid attention to this -- I thought we already did this for all files in the switch to clang-format. Or did you just trigger this for one file?

Reply:
>  thought we already did this for all files in the switch to clang-format.

AFAIK this won't happen unless you are over the limit. So you need to trigger re-flow but creating long lines.

@tamiko might know more.